test/ansible/roles/components_install/tasks/main.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
This role can and should split further. However we should tackle this in the future

Reply:
I would suggest to look at the k8s collection for this one